tiny-menu.el

```diff
@@ -90,11 +90,13 @@ be nil."
           (error (concat "The transition menu specified, \"%s\", is not a valid option, "
                          "check tiny-menu-items.") (or next-menu "N/A")))))))
 
-(defun tiny-menu (&optional menu)
+(defun tiny-menu (&optional menu omit-final-msg-p)
```

Comment:
The `-p` suffix should be reserved for predicates, that is, functions which return a boolean value. See https://www.gnu.org/software/emacs/manual/html_node/elisp/Type-Predicates.html

Instead, this parameter could be called `omit-final-message`, or even simply `silent`.

Reply:
that’s more than fair. ‘silent’ is much better too. I’ve hopefully corrected it, please check.